Unbreak build, otherwise (final) output files are placed into the wrong directory.
--- src/3rdparty/javascriptcore/JavaScriptCore/JavaScriptCore.pri.orig	Tue Nov 11 23:09:41 2014
+++ src/3rdparty/javascriptcore/JavaScriptCore/JavaScriptCore.pri	Tue Nov 11 23:08:32 2014
@@ -7,11 +7,11 @@ CONFIG(standalone_package) {
     isEmpty(JSC_GENERATED_SOURCES_DIR):JSC_GENERATED_SOURCES_DIR = generated
 }
 
-CONFIG(debug, debug|release) {
-    OBJECTS_DIR = obj/debug
-} else { # Release
-    OBJECTS_DIR = obj/release
-}
+#CONFIG(debug, debug|release) {
+#    OBJECTS_DIR = obj/debug
+#} else { # Release
+#    OBJECTS_DIR = obj/release
+#}
 
 INCLUDEPATH = \
     $$PWD \
